shared/oval/cups_disable_browsing.xml

```diff
@@ -11,7 +11,7 @@
       broadcasts and add and configure these printers for immediate use. By
       disabling this browsing capability, the machine will no longer generate
       or receive such broadcasts.</description>
-      <reference source="JL" ref_id="20140306" ref_url="test_attestation" />
+      <reference source="galford" ref_id="20160120" ref_url="test_attestation" />
     </metadata>
     <criteria operator="AND">
       <criterion comment="Ensure remote printer browsing is off"
```

Comment:
The OVAL isn't synced with XCCDF wrt to the requirements. But since the problem isn't in re-using the RHEL/6 OVAL also for RHEL/7, but rather in RHEL-6 implementation itself, I have filed
  https://github.com/OpenSCAP/scap-security-guide/issues/1027

to have this fixed in the future.